docs/src/api/class-page.md

```diff
@@ -494,6 +493,15 @@ page.add_init_script(path="./preload.js")
 await page.AddInitScriptAsync(new PageAddInitScriptOption { ScriptPath = "./preload.js" });
 ```
 
+Note that, if a page uses frames or iframes, the script is added to every frame in the document - to run your script only once, you can test of `window` is equal to `window.top`, for example:
```

Comment:
Also formally incorrect, this is not about `how many times` (once), but about `where` (in the top level frame). So you could say: if you only want to run script inside the main frame of the page (as opposed to child frames), you can check `window === window.top` in the beginning of the script. That way the code runs only in the execution contexts of the main frame.